### PR TITLE
New sample: RasterLayerImageServiceRaster (all .NET SDK's: WPF, UWP, Xamarin.Forms, Android, iOS).

### PR DESCRIPTION
**Pull request by dkemlage**

### CODE REVIEW

**Comment by dkemlage** [comment]
@macca400 Do you have any time for a code review of the new sample: RasterLayerImageServiceRaster?
thanks
-don

File: src/WPF/ArcGISRuntime.WPF.Samples/Samples/Layers/RasterLayerImageServiceRaster/RasterLayerImageServiceRaster.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+            // Load the image service raster
+            await myImageServiceRaster.LoadAsync();
+
+            // Get the service information (aka. metadata) about the image service raster
+            ArcGISImageServiceInfo myArcGISImageServiceInfo = myImageServiceRaster.ServiceInfo;
+
+            // Create a new raster layer from the image service raster
+            RasterLayer myRasterLayer = new RasterLayer(myImageServiceRaster);
+
```
Comment (macca400) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/Layers/RasterLayerImageServiceRaster/RasterLayerImageServiceRaster.xaml.cs]: Could 
```c#
Basemap myBaseMap = myMap.Basemap;
LayerCollection myLayerCollection = myBaseMap.BaseLayers;
myLayerCollection.Add(myRasterLayer);
```
be replaced with 
```c#
myMap.Basemap.BaseLayers.Add(myRasterLayer);
```

File: src/Android/Xamarin.Android/Samples/Layers/RasterLayerImageServiceRaster/RasterLayerImageServiceRaster.cs [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            await myImageServiceRaster.LoadAsync();
+
+            // Get the service information (aka. metadata) about the image service raster
+            ArcGISImageServiceInfo myArcGISImageServiceInfo = myImageServiceRaster.ServiceInfo;
+
+            // Create a new raster layer from the image service raster
+            RasterLayer myRasterLayer = new RasterLayer(myImageServiceRaster);
+
+            // Get the basemap from the map
+            Basemap myBaseMap = myMap.Basemap;
```
Comment (macca400) [inline src/Android/Xamarin.Android/Samples/Layers/RasterLayerImageServiceRaster/RasterLayerImageServiceRaster.cs]: Could 
```c#
Basemap myBaseMap = myMap.Basemap;
LayerCollection myLayerCollection = myBaseMap.BaseLayers;
myLayerCollection.Add(myRasterLayer);
```
be replaced with 
```c#
myMap.Basemap.BaseLayers.Add(myRasterLayer);
```

File: src/Forms/Shared/Samples/Layers/RasterLayerImageServiceRaster/RasterLayerImageServiceRaster.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+            await myImageServiceRaster.LoadAsync();
+
+            // Get the service information (aka. metadata) about the image service raster
+            ArcGISImageServiceInfo myArcGISImageServiceInfo = myImageServiceRaster.ServiceInfo;
+
+            // Create a new raster layer from the image service raster
+            RasterLayer myRasterLayer = new RasterLayer(myImageServiceRaster);
+
+            // Get the basemap from the map
+            Basemap myBaseMap = myMap.Basemap;
```
Comment (macca400) [inline src/Forms/Shared/Samples/Layers/RasterLayerImageServiceRaster/RasterLayerImageServiceRaster.xaml.cs]: Could 
```c#
Basemap myBaseMap = myMap.Basemap;
LayerCollection myLayerCollection = myBaseMap.BaseLayers;
myLayerCollection.Add(myRasterLayer);
```
be replaced with 
```c#
myMap.Basemap.BaseLayers.Add(myRasterLayer);
```

File: src/UWP/ArcGISRuntime.UWP.Samples/Samples/Layers/RasterLayerImageServiceRaster/RasterLayerImageServiceRaster.xaml.cs [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+            await myImageServiceRaster.LoadAsync();
+
+            // Get the service information (aka. metadata) about the image service raster
+            ArcGISImageServiceInfo myArcGISImageServiceInfo = myImageServiceRaster.ServiceInfo;
+
+            // Create a new raster layer from the image service raster
+            RasterLayer myRasterLayer = new RasterLayer(myImageServiceRaster);
+
+            // Get the basemap from the map
+            Basemap myBaseMap = myMap.Basemap;
```
Comment (macca400) [inline src/UWP/ArcGISRuntime.UWP.Samples/Samples/Layers/RasterLayerImageServiceRaster/RasterLayerImageServiceRaster.xaml.cs]: Could 
```c#
Basemap myBaseMap = myMap.Basemap;
LayerCollection myLayerCollection = myBaseMap.BaseLayers;
myLayerCollection.Add(myRasterLayer);
```
be replaced with 
```c#
myMap.Basemap.BaseLayers.Add(myRasterLayer);
```

File: src/iOS/Xamarin.iOS/Samples/Layers/RasterLayerImageServiceRaster/RasterLayerImageServiceRaster.cs [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+            await myImageServiceRaster.LoadAsync();
+
+            // Get the service information (aka. metadata) about the image service raster
+            ArcGISImageServiceInfo myArcGISImageServiceInfo = myImageServiceRaster.ServiceInfo;
+
+            // Create a new raster layer from the image service raster
+            RasterLayer myRasterLayer = new RasterLayer(myImageServiceRaster);
+
+            // Get the basemap from the map
+            Basemap myBaseMap = myMap.Basemap;
```
Comment (macca400) [inline src/iOS/Xamarin.iOS/Samples/Layers/RasterLayerImageServiceRaster/RasterLayerImageServiceRaster.cs]: Could 
```c#
Basemap myBaseMap = myMap.Basemap;
LayerCollection myLayerCollection = myBaseMap.BaseLayers;
myLayerCollection.Add(myRasterLayer);
```
be replaced with 
```c#
myMap.Basemap.BaseLayers.Add(myRasterLayer);
```

**Comment by dkemlage** [comment]
@macca400 Thank you for the review. 

I will make the name changes for the Title as you suggested. I am sorry that I messed that one up. I am new to the common-samples repo and did not realize that we should be getting the name of the sample from the Title of the common-sample issue.

I realize that coding is partly a matter of style. I tend to keep thing verbose, meaning always trying to keep each line of code to creating of a variable so that the user can explicitly see the .NET Runtime type that is being used. I do understand that an intermediate/advance developer will use the chaining style of coding ex: ```myMap.Basemap.BaseLayers.Add(myRasterLayer);``` That being said, you are the reviewer and I will make the change as you suggested.

**Pull request merged**